templates/default/services.conf.erb

```diff
@@ -112,10 +112,13 @@ filter {
         mapping => {"message" => "%{templogdate} %{+templogdate} %{priority} %{logger_name} %{log_message}"}
     }
   } else if [service] == "onlinefs" {
-    # Rest of the log4j services
     dissect {
       mapping => {"message" => "%{templogdate} %{+templogdate} %{priority} %{logger_name}: %{log_message}"}
     }
+    json {
```

Comment:
Can we get the projectId, fgId, fgName and fgVersion from the json? Like before I was parsing the log message: https://github.com/gibchikafa/hopslog-chef/blob/ebb534e02cac84ec0911f7b9025eb6c3ff06a73c/templates/default/services.conf.erb#L57C20-L57C20

Reply:
If we can get that information then also add the tag like here https://github.com/gibchikafa/hopslog-chef/blob/ebb534e02cac84ec0911f7b9025eb6c3ff06a73c/templates/default/services.conf.erb#L58C7-L58C28

Reply:
to get projectId this has to be called `[log_arguments][project_id]` and for others (projectId, fgId, fgName and fgVersion) the names can be found here https://github.com/logicalclocks/clusterj-onlinefs/blob/02018be6284b272d07809c96a1f87c1af6194b16/src/main/java/com/logicalclocks/onlinefs/util/LogArgument.java#L18

as for setting the tag I think I can do it something like this after json extraction:
`
    if [log_arguments][project_id] {
      mutate {
        add_tag => "onlinefs"
      }
    }
`